pkgs/development/libraries/libproxy/default.nix

```diff
@@ -1,82 +1,82 @@
 { lib
-, stdenv
+, bash
+, curl
+, duktape
+, enableDuktape ? lib.meta.availableOn stdenv.hostPlatform duktape
 , fetchFromGitHub
 , fetchpatch
-, pkg-config
-, cmake
-, zlib
-, dbus
-, networkmanager
-, enableJavaScript ? stdenv.isDarwin || lib.meta.availableOn stdenv.hostPlatform duktape
-, duktape
-, pcre
-, gsettings-desktop-schemas
+, gi-docgen
 , glib
-, makeWrapper
-, python3
-, SystemConfiguration
+, gobject-introspection
+, gsettings-desktop-schemas
+, meson
+, ninja
+, pkg-config
+, stdenv
+, vala
 , CoreFoundation
 , JavaScriptCore
+, SystemConfiguration
 }:
 
 stdenv.mkDerivation rec {
   pname = "libproxy";
-  version = "0.4.18";
+  version = "0.5.3";
+
+  outputs = [ "out" "dev" "devdoc" ];
 
   src = fetchFromGitHub {
     owner = "libproxy";
     repo = "libproxy";
     rev = version;
-    hash = "sha256-pqj1LwRdOK2CUu3hYIsogQIXxWzShDuKEbDTbtWkgnQ=";
+    hash = "sha256-qdYB6HJkgboS8kkTvTqLy6Z3JYY5SOJsRl6nZM0iuvw=";
   };
 
-  patches = lib.optionals stdenv.isDarwin [
-    # https://github.com/libproxy/libproxy/pull/189
+  patches = [
     (fetchpatch {
-      url = "https://github.com/libproxy/libproxy/commit/4331b9db427ce2c25ff5eeb597bec4bc35ed1a0b.patch";
-      sha256 = "sha256-uTh3rYVvEke1iWVHsT3Zj2H1F+gyLrffcmyt0JEKaCA=";
+      url = "https://github.com/libproxy/libproxy/pull/261/commits/fe64d1f56b7695bbc8ccb31a34546bc289a4c44d.patch";
```

Comment:
Do we still need this? Expecially, when it is linking to a closed PR.

Reply:
Actually we can just do 

```patch
--- a/pkgs/development/libraries/libproxy/default.nix
+++ b/pkgs/development/libraries/libproxy/default.nix
@@ -32,20 +32,6 @@ stdenv.mkDerivation rec {
     hash = "sha256-qdYB6HJkgboS8kkTvTqLy6Z3JYY5SOJsRl6nZM0iuvw=";
   };
 
-  patches = [
-    (fetchpatch {
-      url = "https://github.com/libproxy/libproxy/pull/261/commits/fe64d1f56b7695bbc8ccb31a34546bc289a4c44d.patch";
-      hash = "sha256-X13rXtmWvssdxy795Ag+2op1vUpEqt6+aicR64zKQnQ=";
-    })
-  ];
-
-  postPatch = ''
-    # prevent meson from installing git hooks...
-    # https://github.com/libproxy/libproxy/issues/262
-    echo -e "#!${lib.getExe bash}\nexit 0" > data/install-git-hook.sh
-    chmod +x data/install-git-hook.sh
-  '';
-
   nativeBuildInputs = [
     gi-docgen
     gobject-introspection
@@ -68,6 +54,14 @@ stdenv.mkDerivation rec {
     gsettings-desktop-schemas
   ]);
 
+  postPatch = ''
+    # Fix running script that will try to install git hooks.
+    # Though it will not do anything since we do not keep .git/ directory.
+    # https://github.com/libproxy/libproxy/issues/262
+    chmod +x data/install-git-hook.sh
+    patchShebangs data/install-git-hook.sh
+  '';
+
   postFixup = ''
     # Cannot be in postInstall, otherwise _multioutDocs hook in preFixup will move right back.
     moveToOutput "share/doc" "$devdoc"
```